.gitignore

```diff
@@ -0,0 +1,17 @@
+__pycache__/
```

Comment:
You included these files here but they are still in your code base. check the tests folder. Please remove the `__pycahe__` folder under your tests folder as well

Reply:
Removed them